app/decorators/controllers/courses_controller_decorator.rb

```diff
@@ -178,6 +183,19 @@ def passing_threshold_group_names
     AssignmentGroup.passing_threshold_group_names
   end
 
+  def relock
```

Comment:
I feel like this is probably better suited to being a model (or model decorator as the case may be) method rather than in the controller. i.e. the JS hits the `/relock` action in the controller, that action is solely responsible for calling `@course.relock_modules` (or whatever you call it) and then the model method handles iterating through the context modules for the course. 

What do you think about refactoring in that way? Is there a reason we put this in the controller instead?

Reply:
Nah I agree, I can move that stuff into the model. There wasn't a reason we put it in here instead, we're still just learning best practices and kinda overlooked it

Reply:
Code has been updated and retested locally (and still functions properly)

Reply:
👍